Bottom: 94084abcdd5c58996302208d4a8baa289d099635
Top:    2e848616f7562d3eb1dca6a8cc51e6875cb66b3e
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-07-02 13:12:36 -0400

Refresh of initial-sound-driver-port

---

diff --git a/drivers/clk/sunxi/clk-sunxi.c b/drivers/clk/sunxi/clk-sunxi.c
index 1e4c98d..0f0de4d 100644
--- a/drivers/clk/sunxi/clk-sunxi.c
+++ b/drivers/clk/sunxi/clk-sunxi.c
@@ -219,7 +219,6 @@ static void sun8i_a23_get_pll1_factors(u32 *freq, u32 parent_rate,
 static void sun7i_get_pll2_factors(u32 *freq, u32 parent_rate,
 				   u8 *n, u8 *k, u8 *m, u8 *p)
 {
-	printk("JDS - sun7i_get_pll2_factors %d\n", *freq);
 	/* we were called to round the frequency, we can now return */
 	if (n == NULL)
 		return;
